speakerbox/src/test/java/com/mapzen/speakerbox/SpeakerboxTest.java

```diff
@@ -124,6 +124,14 @@ public void shouldReturnMuteState() throws Exception {
     }
 
     @Test
+    public void shouldStopOnMute() {
```

Comment:
Your production code changes are most certainly correct. The only issue is that even when I comment out the new lines of code added to `Speakerbox#mute()` this test still passes.

This test relies on `ShadowTextToSpeech` since the real Android `TextToSpeech` class touches audio drivers.

We need to update `ShadowTextToSpeech` so we can verify `stop()` gets called when `isSpeaking()` returns `true` and does not get called when `isSpeaking()` returns `false`.